tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -154,27 +155,26 @@ For that we need samples:
 
 ```julia
 samples = rand(q, 10000);
+size(samples)
 ```
 
 ```julia
-# setup for plotting
-using LaTeXStrings, StatsPlots
-```
-
-```julia
-p1 = histogram(samples[1, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="")
+p1 = histogram(
+    samples[1, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="", ylabel="density"
+)
 density!(samples[1, :]; label="s (ADVI)", color=:blue, linewidth=2)
 density!(samples_nuts, :s; label="s (NUTS)", color=:green, linewidth=2)
 vline!([var(x)]; label="s (data)", color=:black)
 vline!([mean(samples[1, :])]; color=:blue, label="")
 
-p2 = histogram(samples[2, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="")
+p2 = histogram(
+    samples[2, :]; bins=100, normed=true, alpha=0.2, color=:blue, label="", ylabel="density"
+)
 density!(samples[2, :]; label="m (ADVI)", color=:blue, linewidth=2)
 density!(samples_nuts, :m; label="m (NUTS)", color=:green, linewidth=2)
 vline!([mean(x)]; color=:black, label="m (data)")
-vline!([mean(samples[2, :])]; color=:blue, label="")
```

Comment:
Was this line removed on purpose?

Reply:
Sorry, it's a mistakenly removed. Recovered now.

Reply:
Did you forget to push your changes? It seems it's still missing on Github.

Reply:
sorry I was finally testing all blocks - just finished to make a overall commit.